app/routes/questions/$question.tsx

```diff
@@ -16,7 +16,21 @@ export const loader = async ({request, params}: Parameters<LoaderFunction>[0]) =
     throw Error('missing question title')
   }
 
-  return await loadQuestionDetail(request, question)
+  try {
+    return await loadQuestionDetail(request, question)
+  } catch (error) {
+    return {
+      timestamp: new Date().toISOString(),
```

Comment:
I would have preferred to do this later, e.g. [here](https://github.com/StampyAI/stampy-ui/pull/168/files#diff-1f32e8eae48dea5606d6c249c24da8eff4f4d90c2e49dd6764e8cd417f99059cR117), but it complicated things a lot, so I just threw it in here

Reply:
we also need to send the error to the browser, e.g. in additional property `error`, it's much easier to inspect network response than to inspect server logs of cloudflare workers

Reply:
added